tutor/src/screens/task/step/exercise-question.js

```diff
@@ -181,6 +139,25 @@ export default class ExerciseQuestion extends React.Component {
         );
     }
 
+    renderPoints(step) {
+        const points = [step.available_points];
+
+        if (!isNil(step.published_points_without_lateness)) {
+            points.unshift(step.published_points);
```

Comment:
It is a little odd to me that published_points_without_lateness is guarded here when it is published_points that is added, but I suppose they are equivalent in this regard.

Reply:
I thought the same thing, it was also there in the old code, so I assumed it had a reason. Meant to ask you about it 😄